File: jeonghyeon/3week/짝수 홀수 개수.js
function solution(num_list) {
    // filter() : 조건에 맞는 요소들 모아 배열로 반환
    const even = num_list.filter(x => x % 2 == 0).length; // 짝수의 개수
    const odd = num_list.filter(x => x % 2 == 1).length; // 홀수의 개수
    return [even, odd]; // 개수들 배열
}

File: dongmin/week3/짝수 홀수 개수.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];

        answer[0] = (int) Arrays.stream(num_list).filter(n -> n % 2 == 0).count();  //Arrays.stream(num_list)를 통해 배열을 스트림으로 변환.
                                                                                    //filter 메서드를 사용하여 조건을 지정, 
                                                                                    //count 메서드를 사용하여 조건의 맞는 개수를 찾음
        answer[1] = (int) Arrays.stream(num_list).filter(n -> n % 2 != 0).count();

        return answer;
    }
}